docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -884,6 +884,17 @@ spring:
         - Path=/websocket/**
 ----
 
+=== Gateway Metrics Filter
+
+The Gateway Metrics Filter runs as long as the property spring.cloud.gateway.metrics.enabled is not set to false. This filter adds a timer metric named "gateway.requests" with the following tags:
```

Comment:
Put  \` around `spring.cloud.gateway.metrics.enabled` and `false`

Reply:
Done